hellodagger/build.gradle

```diff
@@ -1,24 +1,25 @@
-apply plugin: "com.android.application"
-apply plugin: "kotlin-android"
-apply plugin: "kotlin-android-extensions"
-apply plugin: "kotlin-kapt"
+apply plugin: 'com.android.application'
```

Comment:
I personally prefer double quotes for all strings. the look is consistent and if we ever want to insert arguments we don't have to change from single to double quotes

Reply:
I have no personal taste about it, I've just modified in order to keep string references consistent across Gradle files.

You have Minerva's vote on this :wink:

Reply:
👋 I would keep the double quotes, so you don't need to go from single to double again when you migrate to `.kts` files 😄

Reply:
@sanogueralorenzo agreed, pushed to #364, and rebase #367 over it